images/curl/config/main.tf

```diff
@@ -0,0 +1,21 @@
+variable "extra_packages" {
+  description = "Additional packages to install."
+  type        = list(string)
+  default     = []
+}
+
+module "accts" { source = "../../../tflib/accts" }
+
+output "config" {
+  value = jsonencode({
+    contents = {
+      packages = concat([
+        "curl",
+        "libcurl-rustls4",
+      ], var.extra_packages)
+    }
```

Comment:
is this missing `accounts = module.accts.block`

Reply:
yes

Reply:
```suggestion
    }
    accounts = module.accts.block
```